app/deliver_grant_funding/templates/deliver_grant_funding/grant_setup/description.html

```diff
@@ -0,0 +1,61 @@
+{% extends "common/base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+
+{% block pageTitle %}
+  {{ form.description.label.text }}
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "grants" %}
+
+{% block beforeContent %}
+  {{
+    govukBackLink({
+        "text": "Back",
+        "href": url_for("deliver_grant_funding.grant_setup_name")
+    })
+  }}
+{% endblock beforeContent %}
+
+{% set before_input_text %}
+  <p class="govuk-body">
+    Provide a brief description of the main purpose of the grant. This information will be seen by potential grant recipients.
+  </p>
+  <p class="govuk-hint">
+    Do not include personal information.
+  </p>
+{% endset %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-xl">
+        <span class="govuk-visually-hidden">This section is: </span>Set up the grant
+      </span>
+
+      <form method="post" novalidate>
+        {{ form.csrf_token }}
+
+        <!-- Erroneous-looking fromGroup attribute currently required due to typo in library -->
+        <!-- See here: https://github.com/LandRegistry/govuk-frontend-jinja/blob/45849744d2bf058937402f291e4a2c7a44eb7e23/govuk_frontend_jinja/templates/components/character-count/macro.html#L107 -->
```

Comment:
I think a link to the PR would also be useful to help track 'solved'ness: https://github.com/LandRegistry/govuk-frontend-jinja/pull/96